spec/models/bulkrax/csv_entry_spec.rb

```diff
@@ -18,6 +18,40 @@ module Bulkrax
       end
     end
 
+    describe '#build_metadata' do
+      around do |spec|
+        # Why am I doing this instead of passing it into the parser?  Because of
+        # Bulkrax::ApplicationParser#model_field_mappings.  It uses the bulkrax configuration.
+        old = Bulkrax.field_mappings['Bulkrax::CsvParser']
+        Bulkrax.field_mappings['Bulkrax::CsvParser'] = {
+          "model" => { from: ["model", "work_type"] },
+          "title" => { from: ["title"] },
+          "shape" => { from: ["shape"] },
+          'source_identifier' => { from: ['source_identifier'], source_identifier: true }
+        }
+        class ::Avacado < Work
+          property :shape, predicate: ::RDF::Vocab::DC.format
+        end
+        spec.run
+        Object.send(:remove_const, :Avacado)
+        Bulkrax.field_mappings['Bulkrax::CsvParser'] = old
+      end
+      let(:path) { File.expand_path('../../fixtures/csv/factory_class_test.csv', __dir__) }
```

Comment:
Is there a benefit to using `File.expand_path ...` instead of utilizing [the configured fixture_path](https://github.com/samvera-labs/bulkrax/blob/c3a638139be3ec537aaee250489c7e3c8a4309f3/spec/rails_helper.rb#L72) or [a simple string](https://github.com/samvera-labs/bulkrax/blob/main/spec/parsers/bulkrax/csv_parser_spec.rb#L134)?

Reply:
No benefit, it's what I've carried forward.